src/components/MAPI/releases/ClusterDetailWidgetRelease.tsx

```diff
@@ -159,10 +159,19 @@ const ClusterDetailWidgetRelease: React.FC<
   }, [isAdmin, isImpersonatingNonAdmin, provider, releaseList, releaseVersion]);
 
   const nextVersion = useMemo(() => {
-    return supportedUpgradeVersions.find(
+    if (!releaseList || !releaseVersion) return undefined;
```

Comment:
`supportedUpgradeVersions` contain all upgradable versions (active and inactive), while the `nextVersion` should be determined from only the active versions